functions/go/create-setters/README.md

```diff
@@ -130,12 +166,21 @@ $ kpt fn eval --image gcr.io/kpt-fn/create-setters:unstable --fn-config ./create
 Rendered resource looks like the following:
 
 ```yaml
+# resources.yaml
 apiVersion: v1
 kind: MyKind
 metadata:
   name: foo
 environments: # kpt-set: ${env}
   - dev
   - stage
+role: [stage, dev] # kpt-set: ${env}
 ```
+
+Explanation for the changes:
+- As all the values in `environments` match the setter values of `env`, `# kpt-set: ${env}` comment is added.
+Here, the comment is added to the key node as it is an array node with folded style.
```

Comment:
https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/de2c50a715182f05a0b921a0ef8ab76f10162354/functions/go/create-setters/createsetters/create_setters_test.go#L24

We convert the node to folded style and add comment to the key node correct ?